### DISCUSSION TITLE
How can you set a slide's margin?

### DISCUSSION THREAD

**Discussion by pearl6527** [gitbrent/PptxGenJS#1129] [Q&A]
How can you specify the margins for a slide? Is it possible to do so?

**Comment by gitbrent** [comment]
You'll want to look into [Masters/Templates](https://gitbrent.github.io/PptxGenJS/docs/masters/)

**Comment by vladguan** [comment]
Sorry to resurrect this post but I cannot seem to get slide master margins working. I set the background to a color and would've assumed the color would've stayed within the margin but instead, the whole slide has the background color set.

**Comment by gitbrent** [comment]
That's by design - background color is a slide property and affects the entire slide.
It sounds like you want a shape with a background color that covers almost all of the slide.
